test/UnitTests/MSTest.Analyzers.UnitTests/AvoidExpectedExceptionAttributeAnalyzerTests.cs

```diff
@@ -44,6 +44,276 @@ public class TestClass
             }
             """;
 
-        await VerifyCS.VerifyAnalyzerAsync(code);
+        string fixedCode = """
+            using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+            [TestClass]
+            public class TestClass
+            {
+                [TestMethod]
+                public void TestMethod()
+                {
+                }
+
+                [TestMethod]
+                public void TestMethod2()
+                {
+                }
+
+                [ExpectedException(typeof(System.Exception), AllowDerivedTypes = true)]
+                [TestMethod]
+                public void TestMethod3()
+                {
+                }
+
+                [ExpectedException(typeof(System.Exception), "Some message", AllowDerivedTypes = true)]
+                [TestMethod]
+                public void TestMethod4()
+                {
+                }
+            }
+            """;
+
+        var test = new VerifyCS.Test
+        {
+            TestState =
+            {
+                Sources =
+                {
+                    code,
+                },
+            },
+            FixedState =
+            {
+                Sources =
+                {
+                    fixedCode,
+                },
+                ExpectedDiagnostics =
```

Comment:
Let's link this issue https://github.com/microsoft/testfx/issues/4034 and say the expected diagnostics should be gone when it's fixed.